Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8.11.4 as BUILD
+
+WORKDIR /app
+COPY ./website /app
+COPY ./docs /docs
+
+RUN npm install
+RUN npm run build
+
+ENTRYPOINT ["npm", "start"]
+
```

Comment:
```
FROM node:8.11.4

WORKDIR /app

COPY package.json .

RUN npm install

COPY ./website .
COPY ./docs /docs

RUN npm run build

ENTRYPOINT ["npm", "start"]
```

Reply:
The package.json is in ./website/package.json - did you test this or is it sudo?

Reply:
Sorry updated. Did not test it, just added the improvement.